### PR TITLE
fix(NRQL): fixing left nav

### PR DESCRIPTION
**Pull request by jeff-colucci**

### CODE REVIEW

File: src/nav/nrql.yml [hunk 1]
```diff
@@ -29,40 +29,40 @@ pages:
...
         path: /docs/nrql/nrql-references/nrql-math-using-select
-  - title: Query tools
+  - title: NRQL tools
     pages:
       - title: New Relic query builder
         path: /docs/query-data/nrql-new-relic-query-language/query-tools/chart-builder-create-query-visualizations
       - title: NerdGraph (GraphQL API)
         path: /docs/query-data/nrql-new-relic-query-language/query-tools/nrql-api-queries
-  - title:  Explore NRQL
+  - title: NRQL querying
```
Comment (akristen) [inline src/nav/nrql.yml:38]: `Query NRQL` reads better, even if it eschews the NRQL-as-adjective-noun structure of the other titles.

File: src/nav/nrql.yml [hunk 2]
```diff
@@ -29,40 +29,40 @@ pages:
...
     pages:
-      - title: Set a query's time range
-        path: /docs/nrql/using-nrql/query-time-range
-      - title: Query metric data
-        path: /docs/telemetry-data-platform/get-data/apis/query-metric-data-type
-      - title: Use arrays for organization
+      - title: Arrays
         path: /docs/nrql/using-nrql/arrays-in-nrql
-      - title: Use buckets to segment your data
+      - title: Buckets
```
Comment (akristen) [inline src/nav/nrql.yml:42]: Oh yeah the single words is better. Def like that.

**Review by akristen — APPROVED** [review]
OK! I opened every doc, x-checked that the left nav moved w/ the new doc. everything looks good! The only thing i noticed that should be a fast follow (as it wasnt part of this work) is this doc:

https://deploy-preview-16433--docs-website-netlify.netlify.app/docs/nrql/nrql-syntax-clauses-functions/

In the nav it's called 'nrql syntax' but the doc title is 'nrql reference'. i would rename it to nrql syntax to maintain context, esp bc there's a category called nrql reference now. :) Approving!

**Comment by jeff-colucci** [comment]
I will make a note to do the syntax/reference doc in another ticket, it's part of an upcoming huge rework for that ticket and we'll get it taken care of there. Thanks!

**Pull request merged**